pages/dkp/kommander/2.2/dkp-upgrade/upgrade-kommander/index.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: layout.pug
+navigationTitle: Upgrade Kommander
+title: Upgrade Kommander
+menuWeight: 20
+excerpt: Steps to upgrade Kommander via CLI
+beta: false
+enterprise: true
+---
+ 
+This section describes how to upgrade your Kommander cluster and all its Platform Applications to the latest versions. Kommander is installed on your Management cluster and upgrading it must be the first step you take when upgrading DKP, to prevent compatibility issues.
```

Comment:
```suggestion
This section describes how to upgrade your Kommander Management cluster and all its Platform Applications to the latest versions. Kommander is installed on your Management cluster and upgrading it must be the first step you take when upgrading DKP, to prevent compatibility issues.
```